solr/core/src/test/org/apache/solr/update/processor/FieldMutatingUpdateProcessorTest.java

```diff
@@ -104,14 +104,14 @@ public void testUniqValues() throws Exception {
 
     assertNotNull(d);
 
-    assertEquals(Arrays.asList("Hoss", "Man", "Hoss"), d.getFieldValues("name"));
-    assertEquals(Arrays.asList("Hoss", "Man"), d.getFieldValues("uniq_1_s"));
-    assertEquals(Arrays.asList("Foo", "Hoss", "Man", "Bar"), d.getFieldValues("uniq_2_s"));
-    assertEquals(Arrays.asList(5.0F, 23, "string"), d.getFieldValues("uniq_3_s"));
+    assertEquals(List.of("Hoss", "Man", "Hoss"), List.copyOf(d.getFieldValues("name")));
```

Comment:
why copyOf?    Does the assert some how change the source value?

Reply:
https://errorprone.info/bugpattern/UndefinedEquals

This changes a unknown collection for equality checking into a List which has defined properties.